roles/iridl/templates/docker_compose.j2

```diff
@@ -69,6 +74,9 @@ services:
 
   ingrid:
     image: iridl/ingrid:{{ingrid_version}}
+{% if docker_compose_version == "v2" %}
+    container_name: "{{ compose_project_name }}_ingrid_1"
+{% endif %}
```

Comment:
Actually, I did this because docker_compose_v2 changes the created container names to use - instead of _.  I'm forcing the names so they will be consistent. 

There is another change where I specify a variable for the "docker compose" command.

Reply:
I understand that, but as far as I know the only reason container names matter is because of the `docker exec` commands in the scripts, and I think replacing `docker exec` with `docker compose exec` is something we should do anyway, and if we do that, this change will no longer be necessary.

(But as long as we're maintaining CentOS 7 compatibility, your way is probably simpler. We can defer the change to `docker compose exec`.)

Reply:
Can you do it for all the services, then? Currently you have it so that ingrid and squid have underscores and the rest have dashes. That would be a pain to deal with.

Reply:
done.